packages/device_info/android/src/main/java/io/flutter/plugins/deviceinfo/DeviceInfoPlugin.java

```diff
@@ -77,4 +78,25 @@ public void onMethodCall(MethodCall call, Result result) {
       result.notImplemented();
     }
   }
+
+  //simple emulator detection based on the flutter tools detection
+  //and a couple of legacy detection systems
```

Comment:
Use the classic `/** ... */` JavaDoc style?

Reply:
Jup, in the last time I switched too much between Dart, Kotlin, Swift, Objective-C and Java... I have no idea what's right anymore. :-P

Reply:
Ah, I'll help you out then, since I have our style guides somewhat internalized. I _think_ we don't need this particular method (see comment below), but I'll format the docs in other places in my comments.